examples/README.txt

```diff
@@ -1,3 +0,0 @@
-
-Using dirty_cat
-===============
```

Comment:
This is used as the title for the example page:
https://output.circle-artifacts.com/output/job/8b13cf5a-1866-482f-b0d6-728a852d5100/artifacts/0/doc/auto_examples/index.html

Reply:
You're right, I've put it back and found a way for the README not to appear on the main page (changing the `start_line` in ìnclude`)